### PR TITLE
Update csrankings-9.csv

### PR DESCRIPTION
**Pull request by actbee**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings.csv`._

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings-9.csv [hunk 1]
```diff
@@ -700,6 +700,7 @@ Wenliang Du,Syracuse University,http://www.cis.syr.edu/~wedu,1aMkcasAAAAJ
 Wenpin Jiao,Peking University,http://sei.pku.edu.cn/~jwp,NOSCHOLARPAGE
 Wenping Wang,University of Hong Kong,https://i.cs.hku.hk/~wenping,28shvv0AAAAJ
 Wenqing Zhang,Fudan University,http://homepage.fudan.edu.cn/wqzhang/testete,NOSCHOLARPAGE
+Wenrui Chen,Hunan University,http://eeit.hnu.edu.cn/info/1318/4594.htm,c3bvnWIAAAAJ
```
Comment (emeryberger) [inline csrankings-9.csv:703]: Not in CS; not clearly eligible for inclusion.

File: csrankings-9.csv [hunk 2]
```diff
@@ -1387,6 +1388,7 @@ Yaochun Shen,University of Liverpool,https://liverpool.ac.uk/electrical-engineer
 Yaohang Li,Old Dominion University,http://www.cs.odu.edu/~yaohang,NOSCHOLARPAGE
 Yaohui Jin,Shanghai Jiao Tong University,http://front.sjtu.edu.cn/~jinyh/index.html,H_7_oVcAAAAJ
 Yaoliang Yu,University of Waterloo,https://cs.uwaterloo.ca/~y328yu,zbXIQMsAAAAJ
+Yaonan Wang,Hunan University,http://eeit.hnu.edu.cn/info/1277/4490.htm,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-9.csv:1391]: Not in CS; not clearly eligible for inclusion.

File: csrankings-9.csv [hunk 3]
```diff
@@ -1518,6 +1520,7 @@ Ying Zhu,Ontario Tech University,https://engineering.ontariotechu.ca/people/ecse
 Ying Zou,Queen's University,https://www.ece.queensu.ca/people/Y-Zou,pCYlknMAAAAJ
 Ying-Dar Lin,National Chiao Tung University,http://speed.cis.nctu.edu.tw/~ydlin,6ZzMxhsAAAAJ
 Ying-Ping Chen,National Chiao Tung University,http://www.cs.nctu.edu.tw/~ypchen,5Fb9380AAAAJ
+Ying-Qing Xu,Tsinghua University,http://thfl.tsinghua.edu.cn/info/detailed/428,zSiDAt4AAAAJ
```
Comment (emeryberger) [inline csrankings-9.csv:1523]: Not visibly in CS, same issue as others.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-9.csv (hunk 1 above)
Comment (actbee) [reply csrankings-9.csv:703]: Yes. He is in ECE department but can advise Phd students with (bachelor) background in Computer Science and he is doing Robotics. Not so sure if that is eligible.

File: csrankings-9.csv (hunk 2 above)
Comment (actbee) [reply csrankings-9.csv:1391]: The same as the first one.

File: csrankings-9.csv (hunk 3 above)
Comment (actbee) [reply csrankings-9.csv:1523]: He is in information design school and is doing Graphics and HCI. He can advise students with a previous background in Computer Science. Not so sure if that is eligible.

Comment (actbee) [reply csrankings-9.csv:1523]: If all of them are not eligible then I will close this Pull-Request. Sorry for the inconvenience, thank you!

**Pull request closed without merging**